build.py

```diff
@@ -368,10 +382,10 @@ def main():
 
     # Run the selected tool:
     code = 0
+    # pylint: disable=len-as-condition
     if len(sys.argv) > 0 and sys.argv[1] in DIRECT_TOOLS:
```

Comment:
I think you can change this to `if sys.argv and ...` instead of asking `pylint` to ignore.

Reply:
:+1:

Reply:
Actually this caught a real bug — argv being non-empty only guarantees sys.argv[0] but not sys.argv[1]:
```
$ ./build.py 
Traceback (most recent call last):
  File "./build.py", line 605, in <module>
    main()
  File "./build.py", line 590, in main
    if len(sys.argv) > 0 and sys.argv[1] in DIRECT_TOOLS:
IndexError: list index out of range
```

Sending a PR to fix...